packages/amplication-design-system/src/components/Panel/Panel.tsx

```diff
@@ -19,32 +19,38 @@ export type Props = {
   onClick?: (event: any) => void;
 };
 
-export const Panel = ({
-  panelStyle = EnumPanelStyle.Default,
-  className,
-  children,
-  shadow,
-  style,
-  clickable,
-  onClick,
-}: Props) => {
-  return (
-    <div
-      onClick={onClick}
-      style={style}
-      role={clickable ? "button" : undefined}
-      className={classNames(
-        "amp-panel",
-        className,
-        `amp-panel--${panelStyle}`,
-        { "amp-panel--clickable": clickable },
-        { "amp-panel--shadow": shadow }
-      )}
-    >
-      {children}
-    </div>
-  );
-};
+export const Panel = React.forwardRef(
```

Comment:
I assume it is related to the dnd? I didn't see that we are using it anywhere
also, should we drag the entire panel, or should we add a component to the panel to be dragable?